### DISCUSSION TITLE
apollo- portal 1.3.0 版本 怎么调整namespace配置表单页面右边出现 下拉滚动轴

### DISCUSSION THREAD

**Discussion by FRookie** [apolloconfig/apollo#4462] [Q&A] [answered]
因为配置的数据比较多，导致每次要滑动很久才能到底部，因为一些原因现在还不考虑版本升级，只想修改一下页面样式，请问哪位大神能给点意见。ps：自己尝试修改 div添加 overflow-y：scroll，但是不起作用。。。。蓝瘦

<img width="1185" alt="image" src="https://user-images.githubusercontent.com/33781946/178928379-5a0408ea-dabd-41ef-8379-d11b41fefa25.png">

**Comment by nobodyiam** [comment, marked as answer]
I think #3925 is related.
